main/src/main/java/com/sedmelluq/discord/lavaplayer/source/youtube/DefaultYoutubeTrackDetailsLoader.java

```diff
@@ -114,4 +130,49 @@ private String getUnplayableReason(JsonBrowser statusBlock) {
 
     return unplayableReason;
   }
+
+  private JsonBrowser getTrackInfoFromEmbedPage(HttpInterface httpInterface, String videoId) throws IOException {
+    JsonBrowser basicInfo = loadTrackBaseInfoFromEmbedPage(httpInterface, videoId);
+    basicInfo.put("args", loadTrackArgsFromVideoInfoPage(httpInterface, videoId, basicInfo.get("sts").text()));
+    return basicInfo;
+  }
+
+  private JsonBrowser loadTrackBaseInfoFromEmbedPage(HttpInterface httpInterface, String videoId) throws IOException {
+    try (CloseableHttpResponse response = httpInterface.execute(new HttpGet("https://www.youtube.com/embed/" + videoId))) {
+      int statusCode = response.getStatusLine().getStatusCode();
+      if (statusCode != 200) {
+        throw new IOException("Invalid status code for embed video page response: " + statusCode);
+      }
+
+      String html = IOUtils.toString(response.getEntity().getContent(), Charset.forName(CHARSET));
+      String configJson = DataFormatTools.extractBetween(html, "'PLAYER_CONFIG': ", "});writeEmbed();");
+
+      if (configJson != null) {
+        return JsonBrowser.parse(configJson);
+      }
+    }
+
+    throw new FriendlyException("Track information is unavailable.", SUSPICIOUS,
+            new IllegalStateException("Expected player config is not present in embed page."));
+  }
+
+  private Map<String, String> loadTrackArgsFromVideoInfoPage(HttpInterface httpInterface, String videoId, String sts) throws IOException {
+    String videoApiUrl = "https://youtube.googleapis.com/v/" + videoId;
+    String encodedApiUrl = URLEncoder.encode(videoApiUrl, CHARSET);
+    String url = "https://www.youtube.com/get_video_info?video_id=" + videoId + "&eurl=" + encodedApiUrl +
```

Comment:
The whole reason for this PR is because pbj broke age restricted songs. The code in this PR reverts back to the original way of getting video info when an age restricted video is detected.